packages/components/socialFeed/SocialCard/SocialCardWrapper.tsx

```diff
@@ -55,21 +53,19 @@ export const SocialCardWrapper: FC<{
 
   const { setToastError, setToastSuccess } = useFeedbacks();
   const wallet = useSelectedWallet();
-  const selectedNetworkInfo = useSelectedNetworkInfo();
-  const selectedNetworkId = selectedNetworkInfo?.id || "";
-  const authorNSInfo = useNSUserInfo(localPost.authorId);
-  const [, authorAddress] = parseUserId(localPost.authorId);
+  const networkId = post.networkId;
+  const postNetwork = getNetwork(networkId);
   const {
     banPostProposals,
     isLoading: isLoadingProposals,
     refetch: refetchProposals,
-  } = useGetBanPostProposals(selectedNetworkId);
+  } = useGetBanPostProposals(networkId);
   const [isSendingProposal, setIsSendingProposal] = useState(false);
   const [isExecutingProposal, setIsExecutingProposal] = useState(false);
 
   const banPostProposalTitle = useMemo(() => {
-    return `Ban Post: ${post.identifier}`;
-  }, [post.identifier]);
+    return `Ban Post: ${post.localIdentifier}`;
```

Comment:
Just a note: I see that all other has been changed to : post.id, so maybe this is occasion to change this to => post.localId

Reply:
I wanted to really disambiguate id and local id(entifier)